requirements.txt

```diff
@@ -1,2 +1,4 @@
 ansible==2.3.2
 netaddr==0.7.19
+mock==2.0.0
+ddt==1.1.2
```

Comment:
Woops, thanks!

Reply:
Just wondering how this got here, if you didn't do it intentionally.  You do know about the `subtree` merge strategy in Git, right?  (I don't mean the `git subtree` command, which is unrelated.  I mean `git merge -s subtree`.)

Reply:
I had no clue about the subtree merge strategy, no. I was doing it with the default strategy. The requirements.txt file I believe had a conflict with the one at the root and I just manually fixed it, and left it in there. Did this cause something weird we'll have to undo?

Reply:
No, it probably didn't cause any issues except for this one.  It could just have made your live a lot easier, since it's designed to merge another repo as a subtree into the current repo.  The `git subtree` command is basically just a more user-friendly fronted, and we could actually have used it as well:

    git subtree add --prefix=playbooks/roles/whatever https://github.com/open-craft/whatever master

While `git subtree` also allows to later push back to the original repo, it actually copies the full history into the current repo, and it's completely transparent to users, so this would probably have been the easiest option.  But that's for next time. :)